src/types/hashed.ts

```diff
@@ -1,9 +1,8 @@
-import * as _bcrypt from 'bcrypt';
+import type * as Bcrypt from 'bcrypt';
```

Comment:
I wonder how that worked, if `bcrypt` is not available then this will throw, if it's available then the `try` block below will succeed, so I think it's impossible to enter the `catch` block, which means this can all be simplified. Is that correct?

Reply:
It was still a type-only import before and so it wasn't emitted, it's just much clearer to use `import type` to guarantee that to be the case

Reply:
Oh right! I missed that type-only imports aren't emitted (pretty much the whole point :P)